crypt.go

```diff
@@ -890,9 +890,16 @@ func DeleteLUKS2ContainerKey(devicePath, keyslotName string) error {
 	return nil
 }
 
-// RenameLUKS2Container key renames the keyslot with the specified oldName on
-// the LUKS2 container at the specified path.
-func RenameLUKS2ContainerKey(devicePath, oldName, newName string) error {
+type nonProliferationFlag struct {
+}
+
+// INeedToUseAFeatureThatShouldNotProliferate gives a flag that can authorize
+// some functions that could be dangerous in some context to be used.
```

Comment:
// AllowNonAtomicOperation gives a flag to allow some operations that are not atomic.
// Those can be dangerous depending on usage context, they should be used with care and intentionally
// (that's why the extra hoops).